parserasgeo/features/cross_section.py

```diff
@@ -514,6 +520,76 @@ def cut_line_ratio(self):
             last_pt = pt
 
         return cl_length/length
+    
+    def define_channel_n(self):
+        """
+        Checks if the bank stations are represented in Manning's n (i.e. if Manning's n is defined at the statiosn that correspond with the bank stations)
+        If they are not defined at the bank stations, add Manning's n values at those points.
+        """
+        
+        left_bank = self.bank_sta.left
```

Comment:
left_bank and right_bank are only used in line 532, I see no reason to take up two more lines of code, please consolidate